Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/PluginManager.cpp

```diff
@@ -179,14 +179,27 @@ namespace EMStudio
     }
 
     // find the number of active plugins of a given type
-    size_t PluginManager::GetNumActivePluginsOfType(const char* pluginType) const
+    size_t PluginManager::CalcNumActivePluginsOfType(const char* pluginType) const
     {
         return AZStd::accumulate(m_activePlugins.begin(), m_activePlugins.end(), size_t{0}, [pluginType](size_t total, const EMStudioPlugin* plugin)
         {
             return total + AzFramework::StringFunc::Equal(pluginType, plugin->GetName());
         });
     }
 
+    void PluginManager::RemovePersistentPlugin(PersistentPlugin* plugin)
+    {
+        const auto iterator = AZStd::find_if(m_persistentPlugins.begin(), m_persistentPlugins.end(), [=](const AZStd::unique_ptr<PersistentPlugin>& currentPlugin)
```

Comment:
nit/suggestion: I tend to prefer explicit captures (so this would be `[plugin]`) - not critical here as the function is small but usually preferable to be explicit and not drag in types inadvertently